packages/connect-wallet-modal/src/components/WalletsModal/components/AcceptTermsModal.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Button, Modal } from '@reef-knot/ui-react';
+import styled, { css } from '@reef-knot/ui-react/styled-wrapper';
+import { helpers } from '@reef-knot/web3-react';
+import { Terms, WalletModalConnectTermsProps } from '../../Terms';
+import { CommonButtonsContainer } from '../styles';
+import { useReefKnotContext } from '@reef-knot/core-react';
+
+interface Props {
+  open: boolean;
```

Comment:
Are these keys are always available during initial load? e.g. can any of these props be the result of any promise? There were tons of issues with required props in ui kit and js-sdk before

Reply:
It is being used liked this:
https://github.com/lidofinance/reef-knot/pull/92/files#diff-c344bf0ef02e4726a5821ed1e600e1561257f81344fab675a50c7b9454bbf120R115-R121
Looks like everything is available during initial load.

Reply:
Sure thing, if you think that these values must be provided and there won't be any issues, going to approve